firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -29,13 +29,49 @@ void moveSpaces(int motor[], int spaces, int dir)
 {
   int i;
 
-  digitalWrite(MOTOR_SLEEP, HIGH)
-  digitalWrite(MOTOR_RESET, HIGH)
-  digitalWrite(MOTOR_ENABLE, LOW)
+// How many steps per space
+  float stepsPerSquare = 200; //subject to change, motor turns 1.8degrees/step and 200 steps/full revolution
```

Comment:
For modularity, I would suggest making this a variable that's calculated once in setup. It would use #defines to do so. That way, we can change it easily, since it's a magic number rn. 
I imagine the calculation would be something like:
stepsPerSquare = MILLIMETERS_PER_SQUARE * STEPS_PER_MILIMETER;